core/kt-osrd-sim-infra/build.gradle

```diff
@@ -18,6 +18,7 @@ dependencies {
     // PLEASE ADD AND UPDATE DEPENDENCIES USING libs.versions.toml
     implementation project(':kt-fast-collections')
     implementation project(path: ':osrd-geom')
+    implementation project(path: ':envelope-sim')
```

Comment:
Not sure this is necessary either

Reply:
I think EnvelopeTrainPathUtils.kt depends on 'envelope-sim' because it imports Electrification, so this might be necessary